docs/configuration/service/suricata.rst

```diff
@@ -0,0 +1,105 @@
+.. _suricata:
+
+########
+suricata
+########
+
+:abbr:`suricata` Suricata and VyOS are powerful tools for ensuring network security and traffic management. 
+Suricata is an open-source intrusion detection and prevention system (IDS/IPS) that analyzes network packets in real-time. 
+
+
+Suricata Features
+=================
+
+Intrusion Detection (IDS): Analyzes network traffic and detects suspicious activities, attacks, and malicious traffic.
+
+Intrusion Prevention (IPS): Blocks or modifies suspicious traffic in real-time, preventing attacks before they penetrate the network.
+
+Network Security Monitoring (NSM): Collects and analyzes network data to detect anomalies and identify threats.
+
+Multi-Protocol Support: Suricata supports analysis of various network protocols such as HTTP, FTP, SMB, and many others.
+
+In configuration mode, the commands are as follows:
+
+.. code-block:: none
+
+   vyos@vyos# set service suricata
+   Possible completions:
+   +> address-group        Address group name
+   +  interface            Interface to use
+    > log                  Suricata log outputs
+   +> port-group           Port group name
+
+These commands create a flexible interface for configuring the Suricata service, allowing users to specify addresses, ports, 
+and logging parameters.
+
+After completing the service configuration in configuration mode, the main configuration file suricata.yaml is created, 
+into which all specified parameters are added. Then, to ensure proper operation, the command $ update suricata must be run 
```

Comment:
please use the inline opcmd
https://docs.vyos.io/en/sagitta/documentation.html#opcmd